app/Http/Controllers/CharacterBulkAttachDMEntryController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\AttachDMEntryRequest;
+use App\Models\Character;
+use App\Models\Entry;
+
+class CharacterBulkAttachDMEntryController extends Controller
+{
+    /**
+     * @param \App\Http\Requests\AttachDMEntryRequest $request
+     * @param \App\Models\Character $character
+     * @return \Illuminate\Http\Response
+     */
+    public function update(AttachDMEntryRequest $request, Character $character)
+    {
+        $validated = $request->validated();
+        $entries = Entry::findOrFail($validated['dm_entry_ids']);
+        $character->entries()->saveMany($entries);
```

Comment:
Odd , I wouldn't have expected this to trigger the entry observer, but the tests are passing so i guess it does!

Reply:
I believe it's because the entries are being updated, which trigger the observer checks.